android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -236,6 +238,8 @@ private void initializeUpdateAfterRestart() {
                     // Therefore, deduce that it is a broken update and rollback.
                     rollbackPackage();
                 } else {
+                    // Clear the React dev bundle cache so that new updates can be loaded.
+                    clearReactDevBundleCache();
```

Comment:
offline discussion - can't do this due to bug with android https://code.google.com/p/android/issues/detail?id=52962